src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp

```diff
@@ -209,6 +210,61 @@ void BackgroundQuantitiesCompute<Dim>::function(
       std::move(inverse_metric);
 }
 
+void FaceQuantitiesCompute::function(
+    gsl::not_null<return_type*> result, const Scalar<DataVector>& psi,
+    const Scalar<DataVector>& pi, const tnsr::i<DataVector, Dim>& phi,
+    const tnsr::I<DataVector, Dim>& shift, const Scalar<DataVector>& lapse,
+    const InverseJacobian<DataVector, Dim, Frame::ElementLogical,
+                          Frame::Inertial>& inv_jacobian,
+    const ::ExcisionSphere<Dim>& excision_sphere, const Element<Dim>& element,
+    const Mesh<Dim>& mesh) {
+  const auto direction = excision_sphere.abutting_direction(element.id());
+  if (direction.has_value()) {
```

Comment:
[optional] Though it's probably default constructed to a `std::nullopt`, probably good to explicitly set the result if we aren't on the worltube boundary.

```cpp
if (not direction.has_value()) {
  result->reset();
  return;
}
...
```
Also reduces the indent and makes things less squished.